GooglePlayInstant/Editor/PlayInstantBuildConfiguration.cs

```diff
@@ -50,39 +50,40 @@ private class Configuration
         /// <summary>
         /// Optional field used to prevent removal of required components when building with engine stripping.
         /// <see cref="https://docs.unity3d.com/ScriptReference/BuildPlayerOptions-assetBundleManifestPath.html"/>
+        /// Never null.
```

Comment:
is there a [NonNull] or [NotNull] attribute that's applicable here?

Reply:
Good question. There was a [NotNull] but it's not recommended anymore.
https://stackoverflow.com/questions/30896853/notnull-attribute